### PR TITLE
yugabyte: run-jepsen log file name fix

### PR DESCRIPTION
**Pull request by svarnau**
New test names have slashes, but log file name cannot contain slash.

### CODE REVIEW

**Comment by svarnau** [comment]
Several changes piled up based on testing in sample jenkins job.

File: yugabyte/run-jepsen.py [hunk 1]
```diff
@@ -232,8 +231,16 @@ def parse_args():
         help='Enable clock skew nemesis. This will not work on LXC.')
     parser.add_argument(
         '--concurrency',
-        default='4n',
+        default='5n',
```
Comment (frozenspider) [inline yugabyte/run-jepsen.py]: `single-key-acid` test uses `concurrent-generator`, which requires a concurrency to be a multiple of 2. I suggest keeping default concurrency at `4n`.

File: yugabyte/run-jepsen.py [hunk 2]
```diff
@@ -272,106 +279,99 @@ def main():
...
+            total_test_time_sec += test_elapsed_time_sec
+
+            total_elapsed_time_sec = time.time() - start_time
+            logging.info("Finished running %d tests.", num_tests_run)
+            logging.info("    %d OK, %d Problems (%d Timed-Out)",
+                num_everything_looks_good, num_not_everything_looks_good, num_timed_out_tests)
+            logging.info("    %d exited success, %d exited failure.", num_zero_exit_code,
+                num_non_zero_exit_code)
+            logging.info("Elapsed time: %.1f sec, Test time: %.1f sec, Avg test time: %.1f sec", 
+                total_elapsed_time_sec, total_test_time_sec,
```
Comment (frozenspider) [inline yugabyte/run-jepsen.py]: Weird uppercases (after a comma or a digit), keep them consistent?
Also, is this condensed output still easily readable?

File: yugabyte/run-jepsen.py [hunk 3]
```diff
@@ -272,106 +279,99 @@ def main():
...
+                    "=" * 80, test_index, test_description_str, "=" * 80)
+
+            num_tests_run += 1
+            total_test_time_sec += test_elapsed_time_sec
+
+            total_elapsed_time_sec = time.time() - start_time
+            logging.info("Finished running %d tests.", num_tests_run)
+            logging.info("    %d OK, %d Problems (%d Timed-Out)",
+                num_everything_looks_good, num_not_everything_looks_good, num_timed_out_tests)
+            logging.info("    %d exited success, %d exited failure.", num_zero_exit_code,
```
Comment (frozenspider) [inline yugabyte/run-jepsen.py]: Nit: maybe something like `"%d test out of %d returned non-zero exit code", num_non_zero_exit_code, num_tests_run`?

**Review by frozenspider — COMMENTED** [review]
Reviewed, left a few minor comments

File: yugabyte/run-jepsen.py [hunk 4]
```diff
@@ -234,6 +230,14 @@ def parse_args():
...
         default='4n',
         help='Concurrency to specify, e.g. 2n, 4n, or 5n, where n means the number of nodes.')
+    parser.add_argument(
+        '--workloads',
+        default=','.join(TESTS),
+        help='Comma-seperated list of workloads. Default: ' + ','.join(TESTS))
+    parser.add_argument(
+        '--nemeses',
+        default=' '.join(NEMESES),
+        help='Comma-seperated list of nemeses. Default: ' + ' '.join(NEMESES))
```
Comment (frozenspider) [inline yugabyte/run-jepsen.py]: This is weird - right now tooltip says:
```
  --nemeses NEMESES     Comma-seperated list of nemeses. Default: none kill-
                        tserver kill-master pause-tserver pause-master
                        partition kill,pause,partition
```
So is it comma-separated or space-separated?

**Review by frozenspider — COMMENTED** [review]
Left one more comment

File: yugabyte/run-jepsen.py (hunk 4 above)
Comment (svarnau) [reply yugabyte/run-jepsen.py]: Good point. Was changed to accommodate a jenkins restriction, that no longer applies.

**Review by frozenspider — APPROVED** [review]
LGTM!

File: yugabyte/run-jepsen.py [hunk 5]
```diff
@@ -249,7 +252,7 @@ def main():
     run_cmd(SORT_RESULTS_SH)
 
     start_time = time.time()
-    nemeses = NEMESES
+    nemeses = args.nemeses.split(' ')
```
Comment (frozenspider) [inline yugabyte/run-jepsen.py]: Actually, shouldn't this be `split(',')`?

Comment (svarnau) [reply yugabyte/run-jepsen.py]: Ah, this is the key issue. When do we want to run nemesis one at a time, and when do we want to pass multiple down to the to the test all together? That is the real reason I had space and comma separators.  That is really too subtle. The way jenkins is using this, we should just pass these down, not loop through them.

Instead we only loop through  the workloads.

**Pull request merged**